WebRoot/js/zimbraMail/share/view/htmlEditor/ZmHtmlEditor.js

```diff
@@ -835,6 +835,8 @@ function(id, autoFocus) {
         font_formats : fonts.join(";"),
         fontsize_formats : AjxMsg.fontSizes || '',
 		convert_urls : false,
+		anchor_top: false,
```

Comment:
do we need to fix this in modern ui as well?

Reply:
Yes, it's also reproducible on modern.

Reply:
ok then let's file similar ticket for modern ui and fix there as well